asllib/types.ml

```diff
@@ -848,7 +848,7 @@ let rec base_value loc env t =
   | T_Array _ ->
       Error.fatal_from loc
         (Error.NotYetImplemented "Base value of array types.")
-  | T_Bool -> L_Bool true |> lit
```

Comment:
Why do we have two implementations of `base_value`. Can we reuse the one here for `interpreter.ml`?